src/vs/workbench/contrib/chat/browser/agentSessions/agentSessionViewModel.ts

```diff
@@ -376,9 +370,10 @@ class AgentSessionsCache {
 				status: session.status,
 				archived: session.archived,
 
-				timing: {
-					startTime: session.timing.startTime,
-					endTime: session.timing.endTime,
+				timing: hasKey(session.timing, { created: true }) ? session.timing : {
```

Comment:
Are you doing this to preserve some old data the has the previous shape? I would prefer we would change the storage key simply, I do not think a migration is needed from the previous data.

Reply:
Ok will take a look at invaliding the old cache in this case